### PR TITLE
Ammoening part 1

### PR DESCRIPTION
**Pull request by ProfessorPopoff**
Adds in a bunch of prettier sprites by Infra for our ammo and enables the inventory icon element on them all. 

image

They look great, good job Infra. Now for the rest of them... Later. It'll do for now. I cleaned up wherever I could while in the process. I apologize as I don't have the heart in me tonight to shrink all the big ones left over down to world states, but I might tomorrow. Leaving this up for the mean time.

### CODE REVIEW

File: mojave/effects/spawners/lootdrop/gunloot.dm [hunk 1]
```diff
@@ -22,7 +22,6 @@
 	var/loot4 = list(
 				/obj/item/gun/ballistic/automatic/pistol/ms13/pistol22,
 				/obj/item/ammo_box/magazine/ms13/m22,
-				""
```
Comment (Hekzder) [inline mojave/effects/spawners/lootdrop/gunloot.dm]: Keep the quotes, this applies to the entire file. The gun loot spawner checks for three items, the extra empty spots are filled in by the quotes to minimize potential issues.

**Review by Hekzder — CHANGES_REQUESTED** [review]

File: mojave/items/guns/weapons/energy/energy_pistol.dm [hunk 2]
```diff
@@ -24,6 +24,7 @@
 	base_icon_state = "advlas_pistol"
 	inhand_icon_state = "advlas_pistol"
 	fire_sound = 'mojave/sound/ms13weapons/gunsounds/laspistol/las_pistol_2.ogg'
+	cell_type = /obj/item/stock_parts/cell/ms13/mfc
```
Comment (Hekzder) [inline mojave/items/guns/weapons/energy/energy_pistol.dm:28]: You'll need to make a new ammo_type for the adv. las pistol that has proper energy usage to put it at 20 shots with the amount of energy that the MFC has. Laser pistols are balanced around/intended to have 20 shots.

**Review by Hekzder — CHANGES_REQUESTED** [review]

**Comment by ProfessorPopoff** [comment]
Check and check.

**Review by Hekzder — CHANGES_REQUESTED** [review]
Make the Laser Defender a child of Advanced Las Pistol since I just realized it also uses MFC. You shouldn't have to do anything aside from just make it a child of Advanced. Then this is good to merge if you aren't doing any other work on it.

**Review by Hekzder — APPROVED** [review]
This should be good from what I can see.

**Pull request merged**